src/arm-codegen.c

```diff
@@ -336,7 +336,9 @@ void emit_ph2_ir(ph2_ir_t *ph2_ir)
         }
         interm = __r8;
         /* div/mod emulation */
-        /* Preserve the values of the dividend and divisor */
+        /* Preserve the values of the dividend and divisor. This also keeps
```

Comment:
Because the latest changes preserve the original use of `__stmdb` and `__ldm`, I think the comment doesn't need to be changed.